src/model/CoilWaterHeatingAirToWaterHeatPumpWrapped.cpp

```diff
@@ -326,6 +326,22 @@ namespace model {
         OS_Coil_WaterHeating_AirToWaterHeatPump_WrappedFields::PartLoadFractionCorrelationCurveName);
     }
 
+    // ComponentType CoilWaterHeatingAirToWaterHeatPumpWrapped_Impl::componentType() const {
+    //   return ComponentType::Heating;
+    // }
+
+    // std::vector<FuelType> CoilWaterHeatingAirToWaterHeatPumpWrapped_Impl::coolingFuelTypes() const {
+    //   return {};
+    // }
+
+    // std::vector<FuelType> CoilWaterHeatingAirToWaterHeatPumpWrapped_Impl::heatingFuelTypes() const {
+    //   return {FuelType::Electricity};
+    // }
+
+    // std::vector<AppGFuelType> CoilWaterHeatingAirToWaterHeatPumpWrapped_Impl::appGHeatingFuelTypes() const {
+    //   return {AppGFuelType::HeatPump};  // TODO: openstudio-standards uses Electric
+    // }
```

Comment:
CoilWaterHeatingAirToWaterHeatPumpWrapped is **NOT** an HVACComponent... That's a bit problematic...